docs/src/main/sphinx/release/release-469.md

```diff
@@ -128,7 +128,7 @@
 * Fix table read failures when using the Alluxio file system. ({issue}`23815`)
 * Prevent return of incomplete results by the `table_changes` table function. ({issue}`24709`) 
 * Prevent failures on queries accessing tables with multiple nested partition
-  columns. ({issue}`24709`)
+  columns. ({issue}`24629`)
```

Comment:
Could you use an issue number `24628` instead of PR number?

Reply:
I will change to this:

```suggestion
  columns. ({issue}`24628 `)
```